app/scripts/services/sql.js

```diff
@@ -56,25 +56,31 @@ angular.module('sql', [])
       var status_string = "";
       var stmt_parts = this.stmt.split(' ');
       var cmd = stmt_parts[0].toUpperCase();
-      if (cmd in {'CREATE':'', 'DROP':''}) {
+      if (cmd in {
+        'CREATE': '',
+        'DROP': ''
+      }) {
         cmd = cmd + " " + stmt_parts[1].toUpperCase();
       }
 
       if (this.failed == false) {
-        status_string = cmd + " OK (" + (this.duration/1000).toFixed(3) + " sec)";
+        status_string = cmd + " OK (" + (this.duration / 1000).toFixed(3) + " sec)";
         $log.info("Query status: " + status_string);
       } else {
-        status_string = cmd + " ERROR (" + (this.duration/1000).toFixed(3) + " sec)";
+        status_string = cmd + " ERROR (" + (this.duration / 1000).toFixed(3) + " sec)";
         $log.warn("Query status: " + status_string);
       }
 
       return status_string;
     };
 
 
-    SQLQuery.execute = function(stmt, args) {
-      var data = {'stmt': stmt};
-      if (args != undefined) {
+    SQLQuery.execute = function(stmt, args, errorTrace) {
+      var data = {
+        'stmt': stmt
+      };
+
+      if (args != undefined && !$.isEmptyObject(args) ){
```

Comment:
code style: no space between expression and bracket, however, space between bracket and curly bracket